passiveData/src/androidMain/kotlin/org/sagebionetworks/assessmentmodel/passivedata/recorder/sensor/SensorRecord.kt

```diff
@@ -70,12 +70,13 @@ val sensorRecordModule = SerializersModule {
 }
 
 @Serializable
+@SerialName("FirstRecord")
 data class FirstRecord(
```

Comment:
fyi: these records are called "MarkerRecord" on iOS bc they are used to mark UI transitions. Not necessarily the first record. just happens to also be the first be it marks the ZERO timestamp when the clock is started.